### ISSUE TITLE
Wrap around monitors vertically

### ISSUE THREAD

**Issue by cachai** [MouseUnSnag/MouseUnSnag#27]
First of all I really appreciate this app for the main unsnag option. But I also liked the wrap around function too. When I'm using the trackpad it comes in real handy that I can just go the opposite way to reduce the travel distance(trey icon to start menu for example). It;s just that my current set up is my laptop on the bottom and the secondary monitor above the laptop and I would like to wrap around the displays vertically as well. I've searched for some other freeware options but none of them supports multi monitor setup so they were kinda pointless to me. Feel free to point out if I'm wrong and this app can already do that.

**Comment by patricknelson** [comment]
You are correct, just tested (stacked my monitors in my config) just to double check and it indeed only ever wraps horizontally and not vertically.

@GrantMoyer from what you can tell, would that be easy to implement?

**Comment by pavlik3** [comment]
Is it possoble to add vertical wrapping? I have 2 4k displays and 1 FHD below, so it takes a while to jump from top of any big screens to laptop. With the vertical unsnag it'd be mor convinient.
